_pages/pre-work.md

```diff
@@ -2,15 +2,164 @@
 title: Technical Pre-work
 ---
 
-TODO - fill this out
-
-- Goal: bring technical work to an interview, discuss with interviewer
-- Generally candidates should get the option of "homework" or bringing code to show
-    - homework: more fair (doesn't require tech work that can be shown), but more work
-    - pre-existing work: easier, but not all candidates have it
-- Pre-work shouldn't take more than about 3 hours
-    - Candidates should know this when assigned pre-work, and should reach out if it takes longer (something might be wrong)
-- Each position will have its own pre-work, and a guide to evaluating it
-    - TL should ask when assigning pre-work what language they'll use so we can match to an interviewer with some conversant skill there
-- Review of pre-work should focus on "why did you", "how did you", etc - not technical gotchas
-- Pre-work is "due" 1 biz day before the interview, to give time for the reviewer to review
\ No newline at end of file
+**Goal: bring technical work to an interview. Discuss your code, thought processes, and choices with an interviewer.**
+
+Candidates have two options: complete a coding exercise, or bringing a
+code sample (see the [instructions to candidates](#instructions-to-candidates)
+below for the nitty gritty details).
+
+As an interviewer, you'll receive that code at least a day before your interview
+with the candidate. You should plan to spend an hour or two reading the
+candidate's code, running it, and preparing follow-up questions to ask when you
+interview them.
+
+## Before the interview
+
+Before the interview, prepare by reviewing the code, trying to run it (and the
+tests), and thinking about questions you'll want to ask the candidate during
+the interview (see below).
+
+As you review, remember that candidate only spent about 3 hours on this code;
```

Comment:
It's really minor, but I believe we use AP style, which calls for numbers < 10 to be spelled out.